mobile/src/main/java/org/openhab/habdroid/background/ItemUpdateWorker.kt

```diff
@@ -167,22 +166,20 @@ class ItemUpdateWorker(context: Context, params: WorkerParameters) : CoroutineWo
         taskerIntent: String?,
         hasConnection: Boolean,
         httpCode: Int
-    ): Result {
-        return if (runAttemptCount <= if (isImportant) 3 else 10) {
-            if (showToast) {
-                applicationContext.showToast(R.string.item_update_error_no_connection_retry, Toast.LENGTH_LONG)
-            }
-            Result.retry()
+    ): Result = if (runAttemptCount <= if (isImportant) 3 else 10) {
```

Comment:
I think I'd prefer the old version here. Can probably trick the linter to accept it by factoring out the max run count into a variable, which additionally improves readability.